cypress/integration/examples/pear_spec.js

```diff
@@ -0,0 +1,248 @@
+        {"username":"ConnorAndersonLarson","summary":"this is a summary","email":"cal@gmail.com","language":"PHP","date":"09/18/21","repo":"https://github.com/josharagon/self-care-center","status":"","reviewer":"","id":123},
+        {"username":"Shakikka","summary":"this is a summary","email":"shakika@hotmail.com","language":"Other","date":"02/24/21","repo":"https://github.com/owenhallgren/js-fun-at-the-library","status":"active","reviewer":"Jackson","id":777}
+      ]
+      });
+    cy.get('#1.cancel-button').click()
+    cy.get('#home').click()
+    cy.get('.card-container')
+    .children()
+    .should('have.length', 4)
+  })
+
+})
+
+
+describe('Posting a Review Request', () => {
+  before(() => {
+
+    cy.intercept({
+      method: 'POST',
+      url: 'http://localhost:3003/api/v1/reviews'
+    },
+      {
+        statusCode: 200,
+        body:
+        [
+        {"username":"LumberJackJoe","summary":"I need help!","email":"lumberjackjoe@hotmail.com","language":"Other","date":"02/24/21","repo":"github.com/Joemama","status":"","reviewer":"","id":123456}
+      ]
+      });
+
+    cy.fixture('mockReviews.json')
+    .then(reviewData => {
+        cy.intercept('GET', 'http://localhost:3003/api/v1/reviews', reviewData)
+    })
+    cy.visit('http://localhost:3000/') 
+})
+
+  it('Can post a new review request' , () => {
+    
+    cy.get('#addReq').click()
+    
+    cy.get('select').select('Java')
+
+    cy.get('input').type('github.com/Joemama')
+
+    cy.get('textarea').type('I need help!')
+
+    cy.get('button').last().click()
+
+    cy.get('#home').click()
+
+    cy.get('.card-container')
+    .children()
+    .should('have.length', 5)
+  })
+})
+
+describe('Sad Paths', () => {
+  before(() => {
+    cy.fixture('mockReviews.json')
+    .then(reviewData => {
+        cy.intercept('GET', 'http://localhost:3003/api/v1/reviews', reviewData)
+    })
+    cy.visit('http://localhost:3000/') 
+  })
+
+  it('Should show an appropriate message if no requests are available ina searched langauge', () => {
+    cy.get('#languageFilter').select('C')
+    cy.get('.no-results-message').contains('No reviews available')
+  })
+
+  it('Should show an appropriate message if any of the tables have no data', () => {
+    cy.intercept({
+      method: 'PUT',
+      url: 'http://localhost:3003/api/v1/reviews/cancel/1'
+    },
+      {
+        statusCode: 500,
+        body:
+        [{"username":"JeffShepherd","summary":"this is a summary","email":"mynamejeff@yahoo.com","language":"Python","date":"02/24/21","repo":"https://github.com/JeffShepherd/Rancid-Tomatillos","status":"","reviewer":"","id":1}
+      ]
+      });
+    cy.get('#dashBoard').click()
+    cy.get('#1.cancel-button').click()
+    cy.get('td').contains('no reviews')
+  })
+
+  it('If app fails to fetch data, user will recieve an appropriate error', () => {
```

Comment:
Looks good. Same helper displaying fetch failure used in all API requests, so one test should suffice.